recipes/r-saccharis/pre-link.sh

```
@@ -0,0 +1 @@
+conda config --append channels r
```

Shellcheck

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)